packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/new/new.html

```diff
@@ -1,28 +1,43 @@
-<div data-ovh-alert="app.dedicatedClouds.datacenter.backup.new"></div>
-<p data-translate="dedicatedCloud_datacenter_backup_new_description1"></p>
-<p
-    data-ng-bind-html="::'dedicatedCloud_datacenter_backup_new_description2' | translate"
-></p>
-<strong>
+<div
+    data-ovh-alert="app.dedicatedClouds.datacenter.backup.new"
+></div>
+<div data-ng-if="!$ctrl.backup.isLegacy()">
+    <p data-translate="dedicatedCloud_datacenter_backup_new_description1"></p>
+    <p
+        data-ng-bind-html="::'dedicatedCloud_datacenter_backup_new_description2' | translate"
+    ></p>
     <a
         target="_blank"
         rel="noopener"
         class="oui-link_icon"
         data-ng-href="{{ ::$ctrl.backupTariffUrl }}"
     >
-        <span
-            data-translate="dedicatedCloud_datacenter_backup_new_tariff_url"
-        ></span>
+        <strong>
+            <span
```

Comment:
You can convert the `span` to a `strong` tag directly. The nesting is not required :)